### ISSUE TITLE
wrong lanip index

### ISSUE THREAD

**Issue by aircrack-ng** [aircrack-ng/aircrack-ng#213]
**Reported by anonymous on 20 May 2007 02:32 UTC**

inside dump_write_csv( void ):

```
 fprintf( G.f_txt, "%3d.%3d.%3d.%3d, ",
                 ap_cur->lanip[ap_cur->lanip[1](0],),
                 ap_cur->lanip['''ap_cur->lanip[2](2],)''' );
```

should be

```
 fprintf( G.f_txt, "%3d.%3d.%3d.%3d, ",
                 ap_cur->lanip[ap_cur->lanip[1](0],),
                 ap_cur->lanip[ap_cur->lanip[3](2],) );
```

**Comment by aircrack-ng** [comment]
**Modified by misterx on 20 May 2007 02:32 UTC**

**Comment by aircrack-ng** [comment]
**Comment by misterx on 20 May 2007 02:32 UTC**

Fixed in aircrack-ng/aircrack-ng@1c779769d067a7cc94df0f17be61841cf3c0b70d
